### ISSUE TITLE
IDE0009 reported when raising an event in VB, but there is no way to qualify the event

### ISSUE THREAD

**Issue by RobSiklos** [dotnet/roslyn#19807]
**Version Used**: VS 2017 15.3.26510.0-Preview

**Steps to Reproduce**:

1. Create a new VB.NET class library
2. Enable the {Qualify event access with 'Me'} rule
3. Add the following code:
```
Class C
    Public Event E
    Public Sub M()
        RaiseEvent E
    End Sub
End Class
```

**Expected Behavior**:
IDE0009 should not fire on the `RaiseEvent` line - OR - I should be able to qualify the event symbol.

**Actual Behavior**:
IDE0009 is raised on `E`, and if I try to qualify it with `Me`, I get BC30451 - 'Me' is not declared. It may be inaccessible due to its protection level.

**Comment by CyrusNajmabadi** [comment]
Closing out as we are not planning on any investments in this area.
